docs/package.json

```diff
@@ -12,7 +12,7 @@
   "private": true,
   "scripts": {
     "start": "npm run browser:development",
-    "browser:development": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline --host=",
+    "browser:development": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline",
```

Comment:
I find the hot reloading pretty iffy on our docs anyways -- I haven't been depending on it 😟